extensions/binding/org.eclipse.smarthome.binding.tradfri.test/src/test/java/org/eclipse/smarthome/binding/tradfri/internal/TradfriColorTest.java

```diff
@@ -67,35 +84,51 @@ public void testFromHSBTypeKnownGood1() {
         assertEquals(100, color.hsbType.getSaturation().intValue());
         assertEquals(100, color.hsbType.getBrightness().intValue());
     }
+    
+    @Test
+    public void testFromHSBTypeKnownGood2() {
+        TradfriColor color = TradfriColor.fromHSBType(new HSBType("0,100,1"));
+        assertNotNull(color);
+        assertEquals(2, (int) color.rgbR);
+        assertEquals(0, (int) color.rgbG);
+        assertEquals(0, (int) color.rgbB);
+        assertEquals(45914, (int) color.xyX);
+        assertEquals(19615, (int) color.xyY);
+        assertEquals(2, (int) color.brightness);
+        assertNotNull(color.hsbType);
+        assertEquals(0, color.hsbType.getHue().intValue());
+        assertEquals(100, color.hsbType.getSaturation().intValue());
+        assertEquals(1, color.hsbType.getBrightness().intValue());
+    }
 
     @Test
     public void testConversionReverse() {
         // convert from HSBType
-        TradfriColor color = TradfriColor.fromHSBType(HSBType.GREEN);
+        TradfriColor color = TradfriColor.fromHSBType(new HSBType("120,100,1"));
```

Comment:
the test is just fine w/o your changes. If the HSB(120,100,1) adds another corner case please add a separate test case.